app/views/users/registrations/edit.html.haml

```diff
@@ -28,6 +28,10 @@
     %i (we need your current password to confirm your changes)
     %br/
     = f.password_field :current_password, autocomplete: "off"
+  .field
+    = f.label :avatar
+    %br/
```

Comment:
https://github.com/plataformatec/devise/blob/master/app/views/devise/sessions/new.html.erb
In html br tag is written this way, because br is self closing, i.e you don't have to write <br> </br> 
erb2haml conversion retained this `/` and I copied it from above :)